### PR TITLE
[TH] Clean up Global Moderation Team and update listing

### PR DESCRIPTION
**Pull request by Trigonoculus**
- Improved translation
- Removed English as a spoken language inside GMT members' table

~~Article is still outdated, can be updated (automatically?) after #4462 (adds `/meta/group-info`) is merged~~ Done

### CODE REVIEW

File: wiki/People/The_Team/Global_Moderation_Team/th.md [hunk 1]
```diff
@@ -3,64 +3,63 @@ tags:
...
 | ![][flag_MY] [- [bibitaru] -](https://osu.ppy.sh/users/4482419) | ภาษาจีน และ ภาษามาเลเซีย | ผู้ดูแลการแชท |
 | ![][flag_DE] [- Felix](https://osu.ppy.sh/users/8503985) | ภาษาเยอรมัน | ผู้ดูแลการแชท |
-| ![][flag_US] [abraker](https://osu.ppy.sh/users/4635891) | ภาษาอังกฤษ | ผู้ดูแลฟอรัม |
-| ![][flag_US] [Chaos](https://osu.ppy.sh/users/2628870) | ภาษาอังกฤษ | ผู้ดูแลการแชท, ผู้ดูแลบีทแมพ |
+| ![][flag_US] [abraker](https://osu.ppy.sh/users/4635891) |  | ผู้ดูแลฟอรัม |
+| ![][flag_US] [Chaos](https://osu.ppy.sh/users/2628870) |  | ผู้ดูแลการแชท, ผู้ดูแลบีทแมพ |
 | ![][flag_DE] [Clobohne](https://osu.ppy.sh/users/499343) | ภาษาเยอรมัน | ผู้ดูแลการแชท |
 | ![][flag_AR] [Darksonic](https://osu.ppy.sh/users/570042) | ภาษาสเปน | ผู้ดูแลการแชท, ผู้ดูแลฟอรัม |
-| ![][flag_US] [Death](https://osu.ppy.sh/users/3242450) | ภาษาอังกฤษ | ผู้ดูแลการแชท, ผู้สนับสนุนเทคนิคต่างๆ, ผู้ดูแลบีทแมพ |
+| ![][flag_US] [Death](https://osu.ppy.sh/users/3242450) |  | ผู้ดูแลการแชท, ผู้สนับสนุนเทคนิคต่างๆ, ผู้ดูแลบีทแมพ |
```
Comment (C0NS03L) [inline wiki/People/The_Team/Global_Moderation_Team/th.md:54]:
```suggestion
| ![][flag_US] [Death](https://osu.ppy.sh/users/3242450) |  | ผู้ดูแลการแชท, ผู้สนับสนุนทางเทคนิคต่าง ๆ , ผู้ดูแลบีทแมพ |
```

Apply to all the ones with ผู้สนับสนุนทางเทคนิคต่าง ๆ
Im not sure that it should be "ต่าง ๆ," or "ต่าง ๆ ,"

**Review by C0NS03L — CHANGES_REQUESTED** [review]

File: wiki/People/The_Team/Global_Moderation_Team/th.md (hunk 1 above)
Comment (C0NS03L) [reply wiki/People/The_Team/Global_Moderation_Team/th.md:54]: Actually removing ต่าง ๆ might be a good idea

Comment (Trigonoculus) [reply wiki/People/The_Team/Global_Moderation_Team/th.md:54]:
``` yaml
    support: ผู้สนับสนุนทางเทคนิค
```
Already changed as part of #4462, I didn't touch the table here other than these changes

Comment (C0NS03L) [reply wiki/People/The_Team/Global_Moderation_Team/th.md:54]: Huh neat. I will go ahead and review #4462 when Im free then

File: wiki/People/The_Team/Global_Moderation_Team/th.md [hunk 2]
```diff
@@ -3,117 +3,116 @@ tags:
...
+ทีมผู้ดูแลมีหน้าที่รับผิดชอบด้านสวัสดิการของการแชท ฟอรัม และ ดูแลการมีปฏิสัมพันธ์ภายในชุมชน พวกเขาควรได้รับการติดต่อด้วยคำถามใด ๆ ที่เกี่ยวข้องกับแง่มุมการกลั่นกรองของเกมซึ่งรวมถึง (แต่ไม่จำกัดเฉพาะ) ปัญหาเกี่ยวกับการแชท (เช่น ความเงียบ หรือการล่วงละเมิด) และฟอรัม (เช่น การสแปม หรือความคิดเห็นที่ไม่เหมาะสม)
 
 ## การเป็นสมาชิกทีมผู้ดูแล
 
-การเป็นสมาชิกของทีมผู้ดูแลนั้นต้องการการสนับสนุนที่มีจำนวนมาก สมาชิกที่มีศักยภาพมักจะสังเกตเห็นว่าพวกเขาเต็มใจที่จะช่วยเหลือผู้อื่น กิจกรรมภายในชุมชน และ พฤติกรรมที่ดี ความสามารถในการพูดภาษาอื่นที่ไม่ใช่ภาษาอังกฤษ เป็นที่ต้องการอย่างยิ่ง ผู้สมัครจะถูกพูดคุยภายในทีมเพื่อพิจารณาการเพิ่มศักยภาพของพวกเขาในฐานะสมาชิกในทีม
+การมาเป็นสมาชิกของทีมผู้ดูแลนั้น จำเป็นต้องมีการสนับสนุนอย่างมาก สมาชิกที่มีศักยภาพมักจะถูกสังเกตจากที่ว่าพวกเขาเต็มใจที่จะช่วยเหลือผู้อื่น มีกิจกรรมภายในชุมชน และ พฤติกรรมที่ดี ความสามารถในการพูดภาษาอื่นที่นอกเหนือจากภาษาอังกฤษเป็นที่ต้องการอย่างยิ่ง ผู้สมัครจะถูกพูดคุยภายในทีมเพื่อพิจารณาศักยภาพของพวกเขาในฐานะสมาชิกทีม
```
Comment (C0NS03L) [inline wiki/People/The_Team/Global_Moderation_Team/th.md:22]:
```suggestion
การมาเป็นสมาชิกของทีมผู้ดูแลนั้น จำเป็นต้องมีการสนับสนุนอย่างมาก สมาชิกที่มีศักยภาพมักจะถูกสังเกตจากที่ว่าพวกเขาเต็มใจที่จะช่วยเหลือผู้อื่น มีกิจกรรมภายในชุมชนและ พฤติกรรมที่ดี ความสามารถในการพูดภาษาอื่นที่นอกเหนือจากภาษาอังกฤษนั้นเป็นที่ต้องการอย่างยิ่ง ผู้สมัครจะถูกหารือกันภายในทีมเพื่อพิจารณาศักยภาพของพวกเขาในฐานะสมาชิกทีม
```

File: wiki/People/The_Team/Global_Moderation_Team/th.md [hunk 3]
```diff
@@ -3,117 +3,116 @@ tags:
...
-2. อย่ารายงานทุกสิ่งที่เป็นสิ่งเล็กน้อยในการแชท มันเป็นทักษะที่สำคัญในการรู้ว่าอะไรที่ควรรายงาน หรือ ไม่ควร
-3. อย่าตอบคำถามของผู้อื่นโดยไม่รู้ว่าคำตอบนั้นถูกต้อง หรือ ไม่โดยเฉพาะในฟอรัม ข้อเสนอแนะนั้นก็ดี แต่คนที่เข้าใจผิดมันจะไม่เป็นประโยชน์เลย หากคุณไม่ทราบวิธีแก้ปัญหา ควรให้ผู้อื่นตอบ และ เรียนรู้จากพวกเขา
-4. อย่าหักโหมสิ่งต่าง ๆ ช่วยเหลือเพียงพอดีไม่ใช่จำนวนที่มากเกินไป เรื่องนี้เป็นที่รู้กันดีว่าจะทำให้หมดความอดทน และ ความมุ่งมั่น ทำมันให้ง่ายๆ สิ!
+1. อย่าถาม GMT เกี่ยวกับสถานะของคุณในฐานะผู้สมัคร โดย GMT จะไม่เพิ่มคนที่มีส่วนร่วมเพื่อประโยชน์ของตนเองเท่านั้น การที่ไม่เห็นแก่ตัวเป็นกุญแจสำคัญ
+2. อย่ารายงานทุกสิ่งเล็กน้อยในการแชท เป็นทักษะที่สำคัญที่จะรู้ว่าอะไรที่ควรหรือไม่ควรรายงาน
+3. อย่าตอบคำถามของผู้อื่นโดยไม่รู้ว่าคำตอบนั้นถูกต้องหรือไม่ โดยเฉพาะในฟอรัม ข้อเสนอแนะนั้นก็ดี แต่การทำให้คนเข้าใจผิดนั้นจะไม่เป็นประโยชน์เลย หากคุณไม่ทราบวิธีแก้ปัญหา ควรให้ผู้อื่นตอบ และเรียนรู้จากพวกเขา
+4. อย่าหักโหมสิ่งต่าง ๆ ช่วยเหลือเพียงพอดี ไม่ใช่จำนวนที่มากเกินไป เรื่องนี้เป็นที่รู้กันดีว่าจะทำให้หมดความอดทนและความมุ่งมั่น ทำตัวสบาย ๆ!
```
Comment (C0NS03L) [inline wiki/People/The_Team/Global_Moderation_Team/th.md:40]:
```suggestion
4. อย่าหักโหมเกินไป ช่วยเหลือเพียงพอดี ไม่ใช่จำนวนที่มากเกินไป เรื่องนี้เป็นที่รู้กันดีว่าจะทำให้หมดความอดทนและความมุ่งมั่น ทำตัวสบาย ๆ!
```

File: wiki/People/The_Team/Global_Moderation_Team/th.md (hunk 2 above)
Comment (Trigonoculus) [reply wiki/People/The_Team/Global_Moderation_Team/th.md]: I've reworded the first part of the suggestion. Applied the second part in the newest commit

**Review by C0NS03L — APPROVED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**